addons/website_sale/static/src/scss/website_sale.scss

```diff
@@ -99,7 +95,11 @@ $o-wsale-products-layout-grid-gutter-width: min($grid-gutter-width / 2, $o-wsale
     }
 
     .o_wsale_filmstip_container {
-        &:hover {
+        &.o_browser_is_firefox {
+            scrollbar-color: currentColor $light;
+        }
+
+        &:not(.o_browser_is_firefox):hover {
```

Comment:
I wonder if it's intended to have a custom scrollbar on `website_sale` only? 🤔 
To me, it's not consistent to have a custom scrollbar in the "Filters" off-canvas, but not on other offcanvas like the header menu or in Forum.

The custom scrollbar is part of the frontend design, so it would make sense to have it everywhere in the frontend.
This would probably also simplify the code.

What do you think @stefanorigano ?

| Forum | Header Menu | `website_sale` filters |
|--------|--------|--------|
| ![Capture d’écran 2024-02-06 à 14 49 01](https://github.com/odoo/odoo/assets/80679690/cd605a2c-84d7-4e04-a0d9-85744469f2a1) | ![Capture d’écran 2024-02-06 à 14 49 24](https://github.com/odoo/odoo/assets/80679690/d41b1714-339a-4894-a9de-0add1c6f51a8) | ![Capture d’écran 2024-02-06 à 14 49 37](https://github.com/odoo/odoo/assets/80679690/c1f45566-f326-47b1-a6a4-85c060470c79) |